solidity/contracts/bridge/MovingFunds.sol

```diff
@@ -457,8 +465,13 @@ library MovingFunds {
 
         self.terminateWallet(walletPubKeyHash);
 
-        // TODO: Perform slashing of wallet operators, reward the notifier
-        //       using seized amount, and add unit tests for that.
+        self.ecdsaWalletRegistry.seize(
+            self.movingFundsTimeoutSlashingAmount,
+            self.movingFundsTimeoutNotifierRewardMultiplier,
+            msg.sender,
+            wallet.ecdsaWalletID,
+            walletMembersIDs
+        );
 
         // slither-disable-next-line reentrancy-events
         emit MovingFundsTimedOut(walletPubKeyHash);
```

Comment:
Would adding `walletMembersIDs` to the event make sense?
It would probably be too expensive.

Reply:
I don't think it makes sense.